Stripe/NSError+STPCustomerContext.m

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
The public facing error codes are in StripeError. This is just an internal-only helper to generate an NSError object (similar to the PaymentContext NSError category).